### DISCUSSION TITLE
Copyright header business in the ip policy guidance

### DISCUSSION THREAD

**Discussion by emmanuelbernard** [commonhaus/foundation#18] [Q&A]
https://github.com/commonhaus/foundation-draft/blob/main/governance/ip-policy-guidance.md#1-licensing

I think the foundation should be out of the business of giving these examples.
I've received from RHT legal counter recommendations tot he ones listed for example.

**Comment by emmanuelbernard** [comment]
And funny enough CF does not apply these rules for itself today :)

**Comment by ebullient** [comment]
Nothing in this repo constitutes software. I do include prior art in the MIT license for the project. I also link to the reasons for including copyright statments in that particular document. Note that the headers do not assign copyright to the foundation, copyright is retained by the original authors of the software.

Again.. this is a draft, not an affirmed statement. OpenJS recommends similar. Once we get the bootstrapping council formed, this could get removed. Would depend on the combination of opinion/requirements from all participants.

In some cases, projects coming in will have existing copyright statements in all of their source code, which implies getting permission from those folks, etc.

**Reply by Sanne** [reply]
> which implies getting permission from those folks

I'm not sure what you're suggesting, as I honestly didn't understand what @emmanuelbernard is pointing at above (the link is broken, that doesn't help either), so perhaps ignore me if this is not relevant but please bear in mind that it's not doable to get permission from "all those folks" for some of us ;)

**Comment by ebullient** [comment]
Updated link is here: https://github.com/commonhaus/foundation-draft/blob/main/policies/ip-policy.md#copyright-statements

In practice, I've never seen a case where existing copyright statements were removed. I have seen cases where they were updated (en masse) when copyright was transferred from one company to another. If I had a preference, I would want to ensure (going forward) that after the project is donated, copyright remains with the contributor. To that end, where previous copyright statements exist, you'd need another one (per one of the suggestions) stating that change at donation time. (I think, will clarify with folks that actually know, as IANAL. ;) )

**Comment by ebullient** [comment]
I think we can close this. The revised text I think is pretty clear: "If you want to include Copyright statements in your source, we recommend one of the following.. ". There is no requirement that they be present.

**Reply by gavinking** [reply]
Nice.
